src/i18n/langs/it.js

```diff
@@ -0,0 +1,125 @@
+    key_not_exists: 'La chiave non esiste',
+    collapse_all: 'Comprimi tutti',
+    expand_all: 'Espandi tutti',
+    json_format_failed: 'Analisi Json non riuscita',
+    msgpack_format_failed: 'Analisi Msgpack non riuscita',
+    php_unserialize_format_failed: 'Serializzazione PHP non riuscito',
+    clean_up: 'Pulizia',
+    redis_console: 'Console del Redis',
+    confirm_to_delete_connection: 'Confermare per eliminare la connessione?',
+    connection_exists: 'La configurazione della connessione già esiste',
+    close_to_edit_connection: 'È necessario chiudere la connessione prima di modificare',
+    close_to_connection: 'Confermare per chiudere la connessione?',
+    ttl_delete: 'Impostare TTL<=0 Eliminerà la chiave direttamente',
+    max_page_reached: 'Pagina massima raggiunta',
+    add_new_key: 'Nuova chiave',
+    enter_new_key: 'Immettere prima il nuovo nome della chiave',
+    key_type: 'Tipo di chiave',
+    save: 'Salva',
+    enter_to_search: 'Premere Invio per cercare',
+    export_success: 'Esportazione riuscita',
+    select_import_file: 'Seleziona un file',
+    import_success: 'Importazione riuscita',
+    put_file_here: 'Trascina il file qui o fai clic per selezionare',
+    config_connections: 'Connessioni',
+    import: 'Importare',
+    export: 'Esportare',
+    open: 'Apri',
+    close: 'Chiude',
+    open_new_tab: 'Apri in una nuova scheda',
+    exact_search: 'Ricerca esatta',
+    enter_to_exec: 'Premi Invio per eseguire i comandi, su e giù per cambiare la cronologia',
+    pre_version: 'Versione',
+    manual_update: 'Scaricare Manuale',
+    retry_too_many_times: 'Troppi tentativi di riconnessione. Si prega di controllare lo stato del server',
+    key_to_search: 'Ricerca per parole chiave',
+    begin_update: 'Aggiornare',
+    ignore_this_version: 'Ignora questa versione',
+    check_update: 'Ricerca aggiornamenti',
+    update_checking: 'Alla ricerca di aggiornamenti esistenti, aspetta un attimo...',
+    update_available: 'Nuova versione trovata',
+    update_not_available: 'A sua applicazione è stata l\'ultima versione recente',
+    update_error: 'Aggiornamento non riuscito',
+    update_downloading: 'Scaricando...',
+    update_download_progress: 'Download in corso',
+    update_downloaded: 'Aggiornamento download completato, riavvia la app per favore.\
+    [Tips]: Se stai utilizzando Windows, dopo aver chiuso l\'app, attendi che l\'icona del desktop si aggiorni a uno stato normale (circa 10 secondi), quindi puoi riaprirla',
+    mac_not_support_auto_update: 'Il Mac non supporta l\'aggiornamento automatico, Si prega di <a href="https://github.com/qishibo/AnotherRedisDesktopManager/releases">scaricare</a> e reinstallare manualmente,\
+    o eseguire "brew reinstall --cask another-redis-desktop-manager"\
+    <br><br>🧡se ti è utile, puoi sponsorizzare <a href="https://apps.apple.com/app/id1516451072">AppStore</a>, e AppStore lo aggiornerà automaticamente per te.',
```

Comment:
`<br><br>🧡` what's the char after <br>?

Reply:
I don't understand that.

the characters are the same as "en".

![image](https://user-images.githubusercontent.com/11843655/118593721-74fc1b80-b7a8-11eb-9d85-67b528b2cc3b.png)

Reply:
the char is unvisible in web, but it's right in my locale
![image](https://user-images.githubusercontent.com/5136418/118603500-8e23bd00-b7e6-11eb-92ff-912c164bc294.png)